### ISSUE TITLE
Error when using `importantForAccessibility="no-hide-descendants"` prop with `null/undefined` children

### ISSUE THREAD

**Issue by juhasuni** [microsoft/react-native-windows#12978]
You should be able to reproduce the issue using
```
<View importantForAccessibility="no-hide-descendants">{null}</View>
```

or:
```
<View importantForAccessibility="no-hide-descendants" />
```

The children are mapped using `React.children.map` here:
https://github.com/microsoft/react-native-windows/blob/0784c77e0b5720026796ad76f4f8c680aa5cf468/vnext/src-win/Libraries/Components/View/View.windows.js#L178

Then afterwards it is expected that the return value is an array:
https://github.com/microsoft/react-native-windows/blob/0784c77e0b5720026796ad76f4f8c680aa5cf468/vnext/src-win/Libraries/Components/View/View.windows.js#L196

According to docs however `React.Children.map` returns `null` or `undefined` it the children argument is either one.

**Comment by chrisglein** [comment]
You mention an error. What is the error you see with your repro?

**Comment by juhasuni** [comment]
I don't have a repro yet, this is an old bug we found and fixed back then by patching.

**Comment by juhasuni** [comment]
Coming back to this, you can clearly see from the code that there's a bug there, so I don't assume it requires a repro.

**Comment by juhasuni** [comment]
I can see there was an attempt to fix this in https://github.com/microsoft/react-native-windows/commit/481f89da064d5a33f4b5b79088dc760bd49f6dd2 however `undefined` is not handled in the commit correctly. It should [return early here ](https://github.com/microsoft/react-native-windows/blob/481f89da064d5a33f4b5b79088dc760bd49f6dd2/vnext/src-win/Libraries/Components/View/View.windows.js#L178)also if the `children` is `undefined`.

**Comment by juhasuni** [comment]
Sorry, my bad, looks like it is using loose equality operator `==` versus strict `===` so all good 😃 Don't see much of that these days.
